examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,60 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+https://github.com/openshift/origin
```

Comment:
(ultimately i'll probably want to update this doc to instruct people to use the new cluster up command which would include the imagestreams+template automatically)